apps/filecoin-site/src/app/[locale]/case-studies/[slug]/utils/generateStructuredData.ts

```diff
@@ -0,0 +1,15 @@
+import { PATHS } from '@/constants/paths'
+
+import type { CaseStudy } from '../../types/caseStudyType'
+
+import { generateCaseStudyArticleStructuredData } from '@/case-studies/utils/generateCaseStudyArticleStructuredData'
+
+export function generateStructuredData(data: CaseStudy) {
+  const { title, pageDescription, image } = data
+  return generateCaseStudyArticleStructuredData({
+    path: `${PATHS.CASE_STUDIES.path}/${data.slug}`,
+    headline: title,
+    description: pageDescription,
+    image: image ? image.src : undefined,
```

Comment:
Can we use a default image?

Something from `/case-studies` or otherwise ask creative?